packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -791,6 +809,10 @@ export default {
                 data: { ...this.eventData, validationState },
                 logMethod: this.$logger.logWarn
             });
+
+            this.genericFormErrorMessage = errorCount === 1 ?
+                'There is 1 error in the form.' :
+                `There are ${errorCount} errors in the form.`;
```

Comment:
This needs to be coming from translations. And I'd probably extract this into a computed or similar.